static/src/javascripts/projects/common/modules/onward/tech-feedback.js

```diff
@@ -116,6 +117,10 @@ define([
      * override those at the time the email is sent.
      */
     return function () {
+        detect.adblockInUse.then(function(adblockInUse){
+            adblockBeingUsed = adblockInUse;
```

Comment:
I'm confused I think - because we're calling a promise here this is async, which means you can't guarantee that the `adblockBeingUsed` value is going to be set before its used in the below code?

Small test case, this is equivalent right (note using setTimeout to simulate a click delay)? http://jsbin.com/vunucutovi/edit?js,console

```javascript
var myProm = new Promise(function(resolve){
  return resolve('from the promise');
})

setTimeout(function(){
  var myVar = 'not from the promise';
  
  myProm.then(function(promVal){
    myVar = promVal;
    console.log('in the promise, myVar: ' + myVar)
  })
  
  console.log('after the promise, myVar: ' + myVar);
  
  setTimeout(function(){
    console.log('delay the log, myVar:' + myVar)
  }, 10)
}, 1000)
```

That logs out:

```
"after the promise, myVar: not from the promise"
"in the promise, myVar: from the promise"
"delay the log, myVar:from the promise"
```